webapp/channels/src/actions/integration_actions.test.ts

```diff
@@ -13,6 +15,12 @@ jest.mock('mattermost-redux/actions/users', () => ({
     }),
 }));
 
+interface CustomMatchers<R = unknown> {
+    arrayContainingExactly(stringArray: string[]): R;
+}
+
+type GreatExpectations = typeof expect & CustomMatchers;
```

Comment:
I see we already use this in the codebase, but I feel we can do better. I tried a few things from what I found in Google, but couldn't get it right. I leave here a few links in case you want to give it a try:

https://stackoverflow.com/questions/60227432/how-to-get-a-jest-custom-matcher-working-in-typescript
https://stackoverflow.com/questions/59495104/how-to-let-typescript-know-about-custom-jest-matchers
https://github.com/testing-library/react-testing-library/issues/36#issuecomment-440442300